include/h2/tensor/dist_tensor_base.hpp

```diff
@@ -177,6 +179,27 @@ class BaseDistTensor
    */
   bool is_local_empty() const H2_NOEXCEPT { return local_numel() == 0; }
 
+  /** Output a short description of the tensor. */
+  void short_describe(std::ostream& os) const override
+  {
+    os << "DistTensor<" << TypeName<T>() << ", " << get_device() << ">(";
+    proc_grid().short_describe(os);
```

Comment:
For whatever reason, `operator<<` does not work for `ProcessorGrid` here.

Reply:
```suggestion
    tensor_grid.short_describe(os);
```
Since `proc_grid()` returns by value, it'd be better to just use the reference you already have.